atomics/T1560.001/T1560.001.yaml

```diff
@@ -276,3 +276,41 @@ atomic_tests:
       ls -l #{test_folder}
     cleanup_command: |
       rm -Rf #{test_folder}
+- name: 'Encrypts collected data with AES-256 and Base64'
+  auto_generated_guid: 
+  description: |-
+    An adversary may compress all the collected data, encrypt and send them to a C2 server using base64 encoding. 
+    This atomic test tries to emulate the behaviour of the FLEXIROOT backdoor to archive the collected data. FLEXIROOT typically utilizes AES encryption and base64 encoding to transfer the encrypted data to the C2 server. 
+    In this test, standard zip compression and the OpenSSL library are used to encrypt the compressed data.
+  
+    https://attack.mitre.org/versions/v7/software/S0267/
+  supported_platforms:
+  - linux
+  - macos
+  input_arguments:
+    input_folder:
+      description: Path to the folder used to store the test files
+      type: String
+      default: /tmp/t1560
+    input_file:
+      description: 'Name of the compressed and encrypted files'
+      type: String
+      default: t1560_data
+    enc_pass:
+      description: Password used to encrypt the data
+      type: string
+      default: atomic_enc_pass
+  dependency_executor_name: 
+  dependencies: 
```

Comment:
You can remove these two lines as they are empty.

Reply:
As I moved the creation of `folder` and `file` to dependencies, the lines are no longer empty, so I kept them with the new commands.

Reply:
Thank you